readme.md

```diff
@@ -67,16 +67,15 @@ We're happy to discuss design patterns and learn how you're using Marionette.
 
 ### Key Benefits
 
-* Scalable: applications are built in modules, and with event-driven architecture
+* Scalable: applications built in modules with event-driven architecture
 * Sensible defaults: Underscore templates are used for view rendering
-* Easily modifiable: make it work with your application's specific needs
-* Reduce boilerplate for views, with specialized view types
-* Build on a modular architecture with an `Application` and modules that attach to it
-* Compose your application's visuals at runtime with `View` and `regions`
-* Nested views and layouts within visual regions
-* Built-in memory management and zombie-killing in views and regions
-* Event-driven architecture with `Backbone.Radio`
-* Flexible, "as-needed" architecture allowing you to pick and choose what you need
+* Easily modifiable: works with the specific needs of your application
+* Reduce boilerplate: for all views, including specialized types
+* Create: application visuals at runtime with `Region` and `LayoutView` objects
+* Nested: views and layouts within visual regions
+* Built-in: memory management and zombie-killing for views, layoutViews and regions
+* Event-driven architecture: utilizing the `Backbone.Wreqr.EventAggregator`
```

Comment:
Nope. this is coming over from master.  Someone wanted to reformat the key benefits..  I actually think there'd be benefit in redoing it entirely.. but for now just fixing the issue like you've noted is probably good enough.